### ISSUE TITLE
⌨ Accordion metadata not meeting touch target size

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#13119]
⌨ Accessibility Issue

Accordion metadata buttons do not provide the necessary assistive technology (AT) support for activation. Users relying on touch or alternative input methods may struggle to activate the buttons due to their small size or insufficient spacing.

WCAG 2.1 Criteria

- SC SC 2.5.8 - Target Size (Minimum)
Ensure targets meet a minimum size or have sufficient spacing around them. 
- SC 2.5.5 – Target Size
Interactive elements (such as buttons) should have a target size of at least 44×44 CSS pixels unless they meet an exception.

🦋 Expected Behaviour

Accordion metadata buttons should have more space between the trigger and accordion content.

Users should be able to activate buttons using touch, stylus, or other assistive inputs without accidental activation of adjacent elements.

📋 Steps to Reproduce

1. Login as any user
2. Observe the accordion metadata buttons being too close to the trigger

📸 Screenshot

![Image](https://github.com/user-attachments/assets/a8fdcef1-1652-46d6-a620-a231cb10c482)

🙋‍♀️ Proposed Solution

Increase space between accordion trigger and interactive elements.

✅ Acceptance Criteria

-  Users can activate buttons reliably without unintended actions.
- Buttons meet WCAG guideslines mentions without negatively impacting the UI/UX.
